cmd/eudico/daemon.go

```diff
@@ -193,6 +199,24 @@ func daemonCmd(overrides node.Option) *cli.Command {
 			// Set the metric to one so it is published to the exporter
 			stats.Record(ctx, metrics.LotusInfo.M(1))
 
+                        // FIXME: Consider moving this to an independent optional process
+                        // (so that is not dependent on the main process).
+			if cctx.Bool("export-metrics") {
+				go func() {
```

Comment:
Is this still needed? `eudico-stats`now runs in its own process, right?

Reply:
There might be metrics that cannot be collected by `eudico-stats`, such as response time and stuff like that?